examples/00_getting_started.py

```diff
@@ -0,0 +1,220 @@
+"""
+Getting Started
+===============
+
+The purpose of this guide is to provide an introduction to the functionalities of
+``skrub``, an open-source package that aims at bridging the gap between tabular
+data sources and machine-learning models. Please refer to our
+:ref:`installation_instructions` for installing ``skrub``.
+
+Much of ``skrub`` revolves around vectorizing, assembling, and encoding tabular data,
+to prepare data in a format that machine-learning models understand.
+"""
+
+# %%
+# Downloading example datasets
+# ----------------------------
+#
+# The :obj:`~skrub.datasets` module allows us to download tabular datasets and
+# demonstrate ``skrub``'s features.
+
+# %%
+from skrub.datasets import fetch_employee_salaries
+
+dataset = fetch_employee_salaries()
+employees_df, salaries = dataset.X, dataset.y
+
+# %%
+# Explore all the available datasets in :ref:`downloading_a_dataset_ref`.
+
+
+# %%
+# Generating an interactive report for a dataframe
+# -------------------------------------------------
+#
+# To quickly get an overview of a dataframe's contents, use the
+# :class:`~skrub.TableReport`.
+
+# %%
+from skrub import TableReport
+
+TableReport(employees_df)
+
+# %%
+# You can use the interactive display above to explore the dataset visually.
+
+# %%
+# Easily building a strong baseline for tabular machine learning
+# --------------------------------------------------------------
+#
+# The goal of ``skrub`` is to ease tabular data preparation for machine learning.
+# The :func:`~skrub.tabular_learner` function provides an easy way to build a simple
+# but reliable machine-learning model, working well on most tabular data.
+
+
+# %%
+from sklearn.model_selection import cross_validate
+
+from skrub import tabular_learner
+
+model = tabular_learner("regressor")
```

Comment:
I'm not too sure because here the point is to show that in one line of code we get a model that works on the tabular data. separating the "get a model" part from the "it works" part by dumping the rather big representation of the pipeline and its parameters might distract a bit from that

Reply:
I understand, but at the same time it feels kind of a black box. I don't have a strong opinion on this though.

Reply:
I didn't show it because I didn't want to put emphasis on the hidden complexity in one of the first code blocks of the Getting Started